src/ci/run.sh

```diff
@@ -61,6 +61,12 @@ if [ "$DEPLOY$DEPLOY_ALT" != "" ]; then
   elif [ "$DEPLOY_ALT" != "" ]; then
     RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-llvm-assertions"
   fi
+
+  if [ "$RUST_RELEASE_CHANNEL" = nightly ]; then
+      if [ "$MACOSX_DEPLOYMENT_TARGET" != "" ]; then
+	  RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-lldb"
```

Comment:
Could this be directly added to `.travis.yml`? (the relevant builder executing `--enable-lldb`